docs/glossary.md

```diff
@@ -427,6 +427,12 @@ A Query is a synchronous operation that is used to report the state of a Workflo
 
 Remote data encoding is using your custom Data Converter to decode (and encode) your Payloads remotely through endpoints.
 
+<!-- _Tags: [term](/tags/term), [queries](/tags/queries), [explanation](/tags/explanation)_ -->
+
+#### [Replication Lag](/cloud#replication-lag)
```

Comment:
From looking at https://docs.temporal.io/cloud/multi-region#replication-lag, I believe this is what you want here:

```suggestion
#### [Replication Lag](/cloud/multi-region#replication-lag)
```